app/ports/templates/ports/about.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'ports/layout.html' %}
+
+{% block title %}About | {% endblock %}
+{% block content %}
+    <br>
+    <h3>About</h3>
+    <hr>
+    <div class="row">
+        <div class="col-lg-3 col-md-4 bg-light border border-secondary p-2">
+            <strong>Quick Links</strong>
+            <hr>
+                <ul>
+                    <li><a href="{% url 'stats_faq' %}">Statistics FAQ</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp">Git Repository</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp/tree/master/docs">Documentation</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp/tree/master/docs/api_v1">API</a></li>
+                    <li><a href="https://docs.google.com/document/d/1TqNHm3NrtLDelWqY7P4pZyTMZDJrlsPymFZklzV2t24/edit">Feature Requests</a></li>
```

Comment:
This link is soon about to "expire" and should be converted to some better design document. It can be left there if you want, but I assume it will be gone soon.